waffle-ens/README.md

```diff
@@ -0,0 +1,68 @@
+[![CircleCI](https://circleci.com/gh/EthWorks/Waffle.svg?style=svg)](https://circleci.com/gh/EthWorks/Waffle)
+[![](https://img.shields.io/npm/v/@ethereum-waffle/ens.svg)](https://www.npmjs.com/package/@ethereum-waffle/ens)
+
+![Ethereum Waffle](https://raw.githubusercontent.com/EthWorks/Waffle/master/docs/source/logo.png)
+
+# @ethereum-waffle/ens
+
+A simple ens builder for your blockchain needs.
+
+## Installation
+
+In the current version of waffle (v2.x.x) you will install this package as a dependency of the main waffle package - `ethereum-waffle`.
+
+```
+yarn add --dev ethereum-waffle
+npm install --save-dev ethereum-waffle
+```
+
+If you want to use this package directly please install it via:
+```
+yarn add --dev @ethereum-waffle/ens
+npm install --save-dev @ethereum-waffle/ens
+```
+
+## Feature overview
+
+**NOTE**: You do not need to use this package directly. You can install it through the main package (`ethereum-waffle`) and use it instead.
+
+### ENSBuilder
+
+The `ENSBuilder` class is an excellent solution that can help you create your own ENS domain.
```

Comment:
The `ENSBuilder` class allow you to create your own ENS domain system and domains for testing.